types/bellatrix/execution_payload.yaml

```diff
@@ -45,6 +45,7 @@ Bellatrix:
     allOf:
       - $ref: '#/Bellatrix/ExecutionPayloadCommon'
       - type: object
+        additionalProperties: false
```

Comment:
otherwise the example validation fails, because the each of them have to match exactly one schema
without this, the 4844 paylaod would match all 3 schemas, because additional properties are ignored